README.md

```diff
@@ -33,7 +33,7 @@ yarn local-server
 
 ✅ Run the tests
 ```
-yarn tests
+yarn test
```

Comment:
This is odd, if we look at [the latest readme from master](https://github.com/cityofaustin/ctxfloods/blob/master/README.md) this is already in there.

Any idea why it thinks this is new?